extensions/wikia/ContentReview/controllers/ContentReviewApiController.class.php

```diff
@@ -94,6 +95,31 @@ public function disableTestMode() {
 		$this->makeSuccessResponse();
 	}
 
+	/**
+	 * Update revison request status
+	 *
+	 * @throws BadRequestApiException
+	 * @throws PermissionsException
+	 */
+	public function updateReviewsStatus() {
+		if ( !$this->request->wasPosted()
+			|| !$this->wg->User->matchEditToken( $this->request->getVal( 'editToken' ) )
+		) {
+			throw new BadRequestApiException();
+		}
+
+		$pageId = $this->request->getInt( 'pageId' );
+		$wikiId = $this->request->getInt( 'wikiId' );
+		$status = $this->request->getInt( 'status' );
+
+		if ( !$this->wg->User->isAllowed( 'content-review' ) ) {
```

Comment:
I'd move this above getting the parameters from the request though, since we don't have to get them when the request will fail and they won't be used.